ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixParticipant.java

```diff
@@ -95,27 +95,43 @@ public void initialize(String hostName, int port, List<AmbryHealthReport> ambryH
     }
   }
 
+  private List<String> getSealedReplicas() {
+    HelixAdmin helixAdmin = manager.getClusterManagmentTool();
+    InstanceConfig instanceConfig = helixAdmin.getInstanceConfig(clusterName, instanceName);
+    if (instanceConfig == null) {
+      throw new IllegalStateException(
+          "No instance config found for cluster: \"" + clusterName + "\", instance: \"" + instanceName + "\"");
+    }
+    return ClusterMapUtils.getSealedReplicas(instanceConfig);
+  }
+
+  private boolean setSealedReplicas(List<String> sealedReplicas) {
```

Comment:
I think you could add some docs for this method since there is a similarly named method right after.

Reply:
Sure, added.  Also will move the private methods to the bottom of the class with the public ones on top